ts/features/mixpanel/store/selectors/index.ts

```diff
@@ -0,0 +1,6 @@
+import { GlobalState } from "../../../../store/reducers/types";
+
+const mixpanelFeatureSelector = (state: GlobalState) => state.features.mixpanel;
+
+export const isMixpanelInitializedSelector = (state: GlobalState) =>
+  mixpanelFeatureSelector(state).isMixpanelInitialized;
```

Comment:
```suggestion
  state.features.mixpanel.isMixpanelInitialized;
```
As above. What do you think?

Reply:
It makes sense, I'll update it now! Thank you!